stringfuzz/transformers/fuzz.py

```diff
@@ -25,41 +25,63 @@
     'fuzz',
 ]
 
+def fuzz_char(c):
+
+    # with equal probability: replace, keep, add, or delete a character
+    operation = random.randint(1,4)
+
+    # replace it
+    if operation == 1:
+        return random_text(1)
+
+    # keep it the same
+    if operation == 2:
+        return c
+
+    # add a new character
+    if operation == 3:
+        return c + random_text(1)
+
+    # delete it
```

Comment:
might be more clear to be explicit that this is the 4th choice. Like put in the last "if".

Reply:
I thought about that, but I was out of ideas for non-awkwardly ending the function. The best I could come up with was an `raise ValueError("got an invalid char fuzzing operation")`, but that still felt awkward. What do you think? Maybe we can keep this kind of "default" option, but have it be the "keep it the same" operation instead of "delete it"?

Reply:
Why not an if, elif, elif, else?

Reply:
I've been making it a habit not to have `elif`s after `return`s. Do you think the `raise ValueError ...` option might be ok? That way we can have each `if` explicitly reference an `operation`, and catch all cases of invalid options.

Reply:
Yeah that sounds good to me!